packages/babel-plugin-transform-bigint/src/index.js

```diff
@@ -0,0 +1,41 @@
+import syntaxBigInt from "babel-plugin-syntax-bigint";
+
+export default function({ types: t }) {
+  return {
+    inherits: syntaxBigInt,
+
+    visitor: {
+      Program: {
+        exit(path: NodePath, state: Object) {
+          state.file.addImport("bigint-polyfill", "default", "BigInt");
+        },
+      },
+      BinaryExpression(path) {
+        if (path.node.operator === "+") {
+          if (
+            path.node.left.type === "BigIntLiteral" &&
+            path.node.left.type === "BigIntLiteral"
+          ) {
+            path.replaceWith(
+              t.ExpressionStatement(
+                t.CallExpression(
+                  t.MemberExpression(
+                    t.CallExpression(t.Identifier("_BigInt"), [
```

Comment:
so here is where you can use an API method. This hardcodes it so if someone names another variable _BigInt this would have problems (unlikely but possible)

can use something like `path.scope.generateUidIdentifier("BigInt");`